### PR TITLE
Dildos

### PR DESCRIPTION
**Pull request by Eagle-Eyess**
:cl: EagleEyes
Time to slap bitches once again...
/:cl:

we need them back

### CODE REVIEW

**Comment by Carbonhell** [comment]
critical +1

**Comment by ghost** [comment]
-1 slippery slope

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -832,7 +832,11 @@ Turf and target are separate in case you want to teleport some distance from a t
 GLOBAL_LIST_INIT(can_embed_types, typecacheof(list(
 	/obj/item/stack/rods,
 	/obj/item/pipe,
-	/obj/item/organ/butt))) // Hippie - Because butts aren't sharp or pointy
+	/obj/item/organ/butt,
+	/obj/item/dragon,
```
Comment (ghost) [inline code/__HELPERS/unsorted.dm]: unless can_embed_types works only on strict types, this should be all you need to add

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -832,7 +832,11 @@ Turf and target are separate in case you want to teleport some distance from a t
 GLOBAL_LIST_INIT(can_embed_types, typecacheof(list(
 	/obj/item/stack/rods,
 	/obj/item/pipe,
-	/obj/item/organ/butt))) // Hippie - Because butts aren't sharp or pointy
+	/obj/item/organ/butt,
```
Comment (ghost) [inline code/__HELPERS/unsorted.dm]: https://github.com/HippieStation/HippieStation/tree/master/hippiestation#please-mark-your-changes
this is a multi line change and should be commented to fit as
```
// hippie start
CODE
// hippie stop
```

File: hippiestation/code/game/objects/items/dragon.dm [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
+	return(BRUTELOSS)
+
+/obj/item/dragon/sea
+	name = "Sea Dragon Dildo"
+	desc = "It's damp."
+	icon_state = "seadragon"
+	force = 1
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	throwforce = 2
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+	throw_speed = 4
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	embedding = list("embed_chance" = 30)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: ideally this should be in `/obj/item/dragon` and should be overridden when you want it to be different

File: hippiestation/code/game/objects/items/dragon.dm [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+/obj/item/dragon/canine
+	name = "Canine Dildo"
+	desc = "Taking the phrase \"dogging your mates\" to a whole new level."
+	icon_state = "canine"
+	force = 1
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+	throwforce = 2
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+	throw_speed = 4
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+	embedding = list("embed_chance" = 30)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: ideally this should be in `/obj/item/dragon` and should be overridden when you want it to be different

File: hippiestation/code/game/objects/items/dragon.dm [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+
+/obj/item/dragon/equine
+	name = "Equine Dildo"
+	desc = "Yes, it's the whole horse."
+	icon_state = "equine"
+	force = 1
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/item/dragon` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+	throwforce = 3
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: this is different `/obj/item/dragon` so this doesn't need to be removed

File: hippiestation/code/game/objects/items/dragon.dm [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+	throw_speed = 5
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: this is different `/obj/item/dragon` so this doesn't need to be removed

File: hippiestation/code/game/objects/items/dragon.dm [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+	embedding = list("embed_chance" = 40)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: this is different `/obj/item/dragon` so this doesn't need to be removed

File: hippiestation/code/game/objects/items/dragon.dm [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+/obj/structure/statue/dragon/shelf
+	name = "Dragon Dildo Shelf"
+	desc =  "Built to withstand your collection and your sins."
+	icon = 'hippiestation/icons/obj/dicks.dmi'
+	icon_state = "shelf1"
+
+/obj/structure/statue/dragon/shelf/alt
+	name = "Dragon Dildo Shelf"
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/structure/statue/dragon/shelf` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+	desc =  "Built to withstand your collection and your sins."
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: inherited from `/obj/structure/statue/dragon/shelf` and doesn't need to be defined here

File: hippiestation/code/game/objects/items/dragon.dm [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+	icon_state = null
+	force = 1
+	w_class = 1
+	throwforce = 2
+	throw_speed = 4
+	hitsound = 'hippiestation/sound/misc/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: change this to
`user.visible_message("<span class='suicide'>[user] is shoving [src] down [user.p_their()] throat! It looks like they're trying to commit suicide.</span>")`

File: hippiestation/code/game/objects/items/dragon.dm [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+	return(BRUTELOSS)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dragon.dm]: subjective but this should probably be `return(OXYLOSS)`

**Review by ghost — DISMISSED** [review]
dear lord why
but a lot to go over

**Comment by ghost** [comment]
hoooohhh noooo

-1

**Comment by Eagle-Eyess** [comment]
Oh yea 😈

File: hippiestation/code/game/objects/items/dildo.dm [hunk 19]
```diff
@@ -0,0 +1,43 @@
+/obj/item/dildo
+	name = "Bad Dragon"
+	desc = "You really shouldn't see this.. but if you do... Huzzah.. you found a bug..."
+	icon = 'hippiestation/icons/obj/dicks.dmi'
+	icon_state = null
+	force = 1
+	w_class = 1
+	throwforce = 2
+	throw_speed = 4
+	embedding = list("embed_chance" = 25, "embedded_pain_multiplier" = 1, "embedded_fall_pain_multiplier" = 1, "embedded_impact_pain_multiplier" = 1, "embedded_unsafe_removal_pain_multiplier" = 1)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dildo.dm]: everything after embed_chance unless it's non default doesn't need to be there.

File: hippiestation/code/game/objects/items/dildo.dm [hunk 20]
```diff
@@ -0,0 +1,43 @@
+/obj/item/dildo
+	name = "Bad Dragon"
+	desc = "You really shouldn't see this.. but if you do... Huzzah.. you found a bug..."
+	icon = 'hippiestation/icons/obj/dicks.dmi'
+	icon_state = null
+	force = 1
+	w_class = 1
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dildo.dm]: `w_class = WEIGHT_CLASS_TINY`

File: hippiestation/code/game/objects/items/dildo.dm [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+/obj/item/dildo/canine
+	name = "Canine Dildo"
+	desc = "Taking the phrase \"dogging your mates\" to a whole new level."
+	icon_state = "canine"
+
+/obj/item/dildo/equine
+	name = "Equine Dildo"
+	desc = "Yes, it's the whole horse."
+	icon_state = "equine"
+	w_class = 3
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dildo.dm]: `w_class = WEIGHT_CLASS_NORMAL`

**Review by ghost — DISMISSED** [review]

File: hippiestation/code/game/objects/items/dildo.dm (hunk 19 above)
Comment (Eagle-Eyess) [reply hippiestation/code/game/objects/items/dildo.dm]: its not default

**Comment by Carbonhell** [comment]
@tfw

**Comment by Pyko1** [comment]
much needed +1

**Comment by earthmoonjune** [comment]
https://youtu.be/KGleNri4eWY?t=17s

-1

**Comment by praisenarsie** [comment]
we're at a tie

we need @Dudeinyourbackyard to settle it

**Comment by praisenarsie** [comment]
ok nvm he gave a -1

**Comment by praisenarsie** [comment]
if any other GAYme masters want this, please comment with a +1 and this will be reopened with another needed tiebreaker

**Comment by praisenarsie** [comment]
players can input too i guess

**Comment by biggiOS** [comment]
+1 😂 👌

**Comment by Carbonhell** [comment]
No need to close. It is still being discussed.

**Comment by praisenarsie** [comment]
:rage1:

**Comment by praisenarsie** [comment]
-1 from me even though my opinion doesn't matter :mistake:

**Comment by biggiOS** [comment]
that's gonna be a yes from me dawg

**Comment by bardofrage** [comment]
No, for the sole reason that you made them fit in asses. -1

**Comment by Eagle-Eyess** [comment]
Dildos can no longer fit inside asses

**Comment by ghost** [comment]
image
Erik wants dildos to be able to go up the rear end, provided nothing else can go up there.
And that when you fart, it slaps them across the face with it.

**Comment by lcass** [comment]
No, were trying to be the server that doesn't have weird fetish shit and that won't work if we add weird fetish shit again. We're 1 upping paradise by adding these and that aint a good thing. -1.

**Comment by Eagle-Eyess** [comment]
But lcass this is old code they were already implemented and were hilarious we aren’t going to be like paradise and allow people to erp with them

**Comment by Bubblecum** [comment]
+1 important

**Comment by Pyko1** [comment]
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1
Can I +1 again?
+1

**Comment by lcass** [comment]
They were removed because they were being used for glorified ERP, just because you think it's funny to stick a dildo in someones ass doesn't make it not ERP and is literally going 1 step further than just allowing ERP. You are actually providing an Item which sole use is Erping.

**Comment by Eagle-Eyess** [comment]
it cant even fit inside the ass anymore you can only just slap people with it

**Comment by Manly-Dwarf** [comment]
nah if we denying poo for crossing a line, we're denying dildos for crossing a line.

**Comment by TrickyDickRichard** [comment]
poo gives everyone diseases and makes monkeys the hardest mob to kill

**Comment by Maroonenahmoron** [comment]
It only gets a +1 from me if it's not spawned in the map by default, but as an off-station "treasure" or adminspawnable item

**Comment by lcass** [comment]
Maroon has a good comprimise, if it's more of an easter egg thing that you have to find out in space or something I'd be fine with that.

**Comment by Eagle-Eyess** [comment]
That is not one of the things I’d be willing to get rid of. I already have some fun ideas of where to place them on the station station

**Comment by Eagle-Eyess** [comment]
I honestly don’t know why so many people are against this. This is a meme item with a high embed rate that plays a funny ogg sound. It can’t fit in the bum bum anymore so there’s not going to be a lot erp if any at all. If people were going to erp they would do it now with the tons of other items in game that would work just the same, probably even better, like pens for example.

**Comment by JohnGinnane** [comment]
@Eagle-Eyess just to play devil's advocate:
>If people were going to erp they would do it now with the tons of other items in game that would work just the same

The difference is dildos have a sexual aspect associated with them, it's more obvious if someone is using it to attack people, rather than using a pen

**Comment by bardofrage** [comment]
Now that dildos can no longer fit into asses, +1.

**Comment by deathride58** [comment]
These sprites are unironically better than Citadel's dildo sprites. I hope you degenerates are all proud of yourselves.

**Comment by tortellinitony** [comment]
-1

**Comment by dreamcrab** [comment]
if dildos dont fit in asses regardless of size then i could be swayed

are they off-station or maint loot? because i dont want a ton of dildos every round

**Comment by Eagle-Eyess** [comment]
they dont fit in asses since they are normal size and i was thinking of placing about 7 - 10 dildos a most on the station

**Comment by Eagle-Eyess** [comment]
actually 7 - 10 sounds like way too many now that i think about it. Instead only 3, 1 for each type.

**Comment by Carbonhell** [comment]
Heads: erik +1
GMs: +2/3 teamstab/looping
This applies to an implementation with dildos being strictly off station, and just 1 of each,they should be rare
That said, if it doesn't reach a total +2, it will be denied sadly.

**Comment by Lxtoofonea** [comment]
+1

**Comment by Carbonhell** [comment]
@dreamcrab

**Comment by Carbonhell** [comment]
As for now, the code makes them only admin spawnable. Thus, merging it

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**